src/main/resources/css/tracktable.css

```
@@ -40,17 +40,15 @@
     -fx-background-color: rgb(99, 255, 109);
 }
 
-.table-row-cell:notInDisk,
-.table-row-cell:notInDisk:unplayable {
-    -fx-border-color: red;
-    -fx-border-width: 0.5px;
-    -fx-border-style: solid inside;
+/* Unplayable rows: dimmed gray text, no row borders */
+.table-row-cell:unplayable .text {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":unplayable" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)